src/pages/Playlists/Playlists.js

```diff
@@ -0,0 +1,132 @@
+import { Fragment, useState } from "react";
+import { usePlaylists } from "./playlists-context";
+import { DELETE_PLAYLIST, EDIT_PLAYLIST_NAME } from "./playlists-reducer";
+import { CloseButton } from "../../components/CloseButton";
+import { BaseCard } from "../../components/BaseCard";
+import { EditIcons } from "../../components/EditIcons";
+import { useAxios } from "../../custom hooks/useAxios";
+import { API_PLAYLISTS } from "../../urls";
+import { LoadingIndicator } from "../../components/LoadingIndicator";
+import { useNavigate } from "react-router";
+
+export function Playlists() {
+  const { playlists } = usePlaylists();
+  const navigate = useNavigate();
+  return (
+    <div className="container">
+      <h1>All Playlists</h1>
+      {playlists.map((playlist) => {
+        const { _id, name, videos, defaultPlaylist } = playlist;
+        return (
+          <Fragment key={_id}>
+            <PlaylistHeader
```

Comment:
You can create a playlist card component and extract this out. The main page structure would look more cleaner.

Reply:
A playlist is a group of videos and does not have a card as such. So, I made the code readable by breaking it into more components. Thanks for noticing this.